node/node.go

```diff
@@ -210,6 +210,11 @@ func (n *Node) GetAllIssuedESDTs(tokenType string, ctx context.Context) ([]strin
 		return nil, ErrMetachainOnlyEndpoint
 	}
 
+	return n.baseGetAllIssuedESDTs(tokenType, ctx)
+}
+
+// baseGetAllIssuedESDTs returns all the issued esdt tokens
```

Comment:
Can delete this, we only add comments for exported functions. If the func name doesn't add any value to the func description, it can be removed